### ISSUE TITLE
[✅ASSIGNED] Builder InitParams: API_URL and UI Kit

### ISSUE THREAD

**Issue by app-generator** [app-generator/free-site-builder#79]
> `STATUS`:  **ASSIGNED** to @umarali 
---

### Context

In the current version, the URL of the backend and the UI Kit are both hardcoded.
The  `DNDBuilder.setup()` helper needs to be updated to allow the injection of the mentioned variable at runtime:

```javascript
  <script type="module">
    import { DNDBuilder } from "./index";

    DNDBuilder.setup(BACKEND_URL='https://components-server.onrender.com/', UI_KIT='bs5');
    // Defaults:
    // BACKEND_URL=https://components-server.onrender.com/
    // UI_KIT=bs5
    DNDBuilder.render();
  </script>
```

The above variables are used to query the backend server, currently managed via ENV.

https://github.com/app-generator/free-site-builder/blob/main/src/main.ts

```javascript
const BACKEND_URL = "https://components-server.onrender.com/";
const url = import.meta.env.VITE_BACKEND_URL
  ? import.meta.env.VITE_BACKEND_URL
  : BACKEND_URL;

const UI_KIT = "bs5";
const ui_kit = import.meta.env.VITE_UI_KIT
  ? import.meta.env.VITE_UI_KIT
  : UI_KIT;
```

@umarali please take a look.

TY!

**Comment by umarali** [comment]
I'd be glad to hear some feedback on the commits under branch `fix-79 ` ([3c57899](https://github.com/app-generator/free-site-builder/commit/3c578996313f8c5a81eb04f97d8f7e5888dc7f72)). I'm sure there could be a better way.

**Comment by app-generator** [comment]
At first look works great. 
More tests tmrw. 
TY @umarali
